ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -106,6 +106,11 @@ tenants:
         # If vrf_vni > 10000 make sure to adjust mac_vrf_vni_base accordingly to avoid overlap.
         vrf_vni: < 1-1024 >
 
+        # VRF ID | Required.
```

Comment:
I don't think we need to specify this as required. I would suggest rewording:
```yaml
# VRF ID | Optional (required if "vrf_vni" is not set)
# vrf_id is used as default value for "vrf_vni" and "ospf.process_id" unless those are set.
vrf_id: < 1-1024 >

Reply:
Fixed.